### ISSUE TITLE
Issue with <Tailwind> in Tests

### ISSUE THREAD

**Issue by dcramer** [shellscape/jsx-email#201]
- Component or Package Name: jsx-email
- Component or Package Version: 1.0.11
- `@jsxp-email/cli` Version?: n/a
- Operating System (or Browser): mac/linux
- Node Version: 20.12.1

Let me start by apologizing for how annoying this ticket is going to be.

I'm trying to switch from mjml to jsx-email, and it mostly seemed to be going ok, until I ran the test suite:

https://github.com/dcramer/peated/pull/196

The error itself, as far as I can tell, is completely unrelated to jsx-email (I at least surface-level checked deps/code):

```
Error: The locale data for 'person.prefix' aren't applicable to this locale.
  If you think this is a bug, please report it at: https://github.com/faker-js/faker
  v ../../node_modules/@faker-js/faker/dist/esm/chunk-GTEBSQTL.mjs:1:5252
  Object.get ../../node_modules/@faker-js/faker/dist/esm/chunk-GTEBSQTL.mjs:4:205
  stringifyObject ../../node_modules/hash-it/src/stringify.ts:226:49
  stringifyRecursiveAsJson ../../node_modules/hash-it/src/stringify.ts:115:42
  stringifyComplexType ../../node_modules/hash-it/src/stringify.ts:37:12
  stringify ../../node_modules/hash-it/src/stringify.ts:254:12
  stringifyObject ../../node_modules/hash-it/src/stringify.ts:226:34
  stringifyRecursiveAsJson ../../node_modules/hash-it/src/stringify.ts:115:42
  stringifyComplexType ../../node_modules/hash-it/src/stringify.ts:37:12
  stringify ../../node_modules/hash-it/src/stringify.ts:254:12
  ```
  
That error is triggered by running `render(<EmailTemplate />)`, and specifically I've determined that it only occurs when the `<Tailwind>` component is utilized.

What I do know:

- faker is not referenced anywhere in my email generation code
- the test fixtures which use it, they pass in a clean object with no references to faker
- afaict this is not another test leaking over - I've run this one thats failing in isolation
- this test worked fine previously, with the only changing being swapping to jsx-email from mjml

I'm not sure this is a problem with jsx-email, but I'm hoping someone might have an ah-ha moment and find something I missed, or faker truly is being used and somehow this error is exposing itself.

Here's the props being passed to the template:

```
comment={
  id: 1,
  tastingId: 2,
  comment: '**An Comment** on _life_',
  createdAt: 2024-07-25T15:35:16.892Z,
  createdById: 2,
  createdBy: {
    id: 2,
    username: 'percy26565',
    email: 'percy26565_example@gmail.com',
    passwordHash: null,
    pictureUrl: null,
    private: false,
    active: true,
    admin: false,
    mod: false,
    notifyComments: true,
    createdAt: 2024-07-25T15:35:16.784Z
  },
  tasting: {
    id: 1,
    bottleId: 1,
    tags: [ 'demanding', 'suspicious', 'dense', 'whole' ],
    color: null,
    rating: 3.544034754857421,
    imageUrl: null,
    notes: 'Ademptio damnatio bos aestus certus.',
    servingStyle: null,
    friends: [],
    flightId: null,
    comments: 0,
    toasts: 0,
    createdAt: 2024-07-25T15:35:16.828Z,
    createdById: 3,
    bottle: {
      id: 1,
      fullName: 'Murray, Grimes and Pollich Proactive Portals #92',
      name: 'Proactive Portals #92',
      uniqHash: 'f5d90418c168008f1ac03a932e18a7e7',
      searchVector: "'92':7A,10B 'grime':2A,12B 'murray':1A,11B 'pollich':4A,14B 'portal':6A,9B 'proactiv':5A,8B",
      category: null,
      brandId: 1,
      bottlerId: null,
      statedAge: null,
      flavorProfile: null,
      vintageYear: null,
      caskSize: null,
      caskType: null,
      caskFill: null,
      releaseYear: null,
      description: null,
      descriptionSrc: null,
      tastingNotes: null,
      suggestedTags: [],
      avgRating: null,
      totalTastings: 0,
      createdAt: 2024-07-25T15:35:16.800Z,
      updatedAt: 2024-07-25T15:35:16.800Z,
      createdById: 5
    },
    createdBy: {
      id: 3,
      username: 'judd.harris4075',
      email: 'joe@example.com',
      passwordHash: null,
      pictureUrl: null,
      private: false,
      active: true,
      admin: false,
      mod: false,
      notifyComments: true,
      createdAt: 2024-07-25T15:35:16.786Z
    }
  }
}
baseUrl=http://localhost:3000
```

**Comment by shellscape** [comment]
happy to help if you follow our issue template.

**Comment by dcramer** [comment]
I've added the version information, but everything is current.

**Comment by shellscape** [comment]
Please see the section about a reproduction. It was annotated with importance.

**Comment by dcramer** [comment]
Unfortunately the amount of effort to reproduce this outside of my repro (let alone via a third party service) is likely very large, given it could be related to any number of things in the cascade of dependencies.

**Comment by shellscape** [comment]
That's precisely why we ask for a reproduction - it's completely unfair to offload the effort of isolating an obscure error to the maintainers of an open source project. 

We'll be here to help if you'd like create a reproduction we can work with.

**Comment by dcramer** [comment]
The reason I opened the ticket is because you - the maintainers - might have some obvious insight, that I otherwise wouldn't given I didn't build the library, know the internals, etc. Ultimately if I could TDD this problem away I already would have, given I have a repro in my own codebase. I understand reproductions are time consuming, and if thats the requirement to get any level of information (which is totally your choice), I'll solve this myself or not use jsx-email.

**Comment by shellscape** [comment]
You've been around a _long_ time in the ecosystem. You're the CTO of a successful company that not only contributes considerable amounts of money to open source, but also maintains open source. When one runs across the following in an issue:

>   ⚡️ katchow! We 💛 issues.
> 
>   🚨 Your issue will be CLOSED if:
>      - This template is removed
>      - Parts of this template are removed
> 
>   👉🏽 Need help or tech support? Please don't open an issue!
>   Head to https://discord.gg/FywZN57mTg or start a Discussion https://github.com/shellscape/jsx-email/discussions

... the expectation there is it'll be read and followed. There are a ton of visual cues there to indicate importance. There's really not much reason for overlooking that. 

We've got two _really really good_ channels for general Q & A, discussion, and "hey has anyone seen this bananas stuff?" type questions. The clip from the template above includes them both. I'm personally always in Discord, and the discussions actually have a good amount of interactions given the smaller userbase we've got.

So, if you want to open a bug report, follow the dang instructions. If you want some casual chat about what might be going on and aren't expecting anyone to deep dive, you've got two options.
